md/cfcli/apps.md

```diff
@@ -0,0 +1,33 @@
+# Organizations and Spaces
+
+The applications and users of a Cloud Foundry platform are split into Organizations and Spaces. This gives a way to built a multi-tenant environment with minimal headache since each of these organizations and spaces can be given it's own resource quotas and access controls.
+
+As part of your free trial, you have administrator rights for your own organization with a few different spaces by default. By default, there are a few applications running in the "sample" space (selected at login).
+
+To check which apps are running, run:
```

Comment:
From here we're not really talking about orgs and spaces anymore. Break this out into a "useful CLI commands" section?

Reply:
Agreed. I started on one topic and then decided that a few other things flowed in the same area. I'll break it up into multiple files